api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -120,68 +107,81 @@
     this.userService = userService;
     this.userRecentResourceService = userRecentResourceService;
     this.userDao = userDao;
+    this.leonardoMapper = leonardoMapper;
     this.clock = clock;
   }
 
-  private Stream<org.pmiops.workbench.leonardo.model.ListClusterResponse> filterByClustersInList(
-      Stream<org.pmiops.workbench.leonardo.model.ListClusterResponse> clustersToFilter,
-      List<String> clusterNames) {
+  private Stream<org.pmiops.workbench.leonardo.model.ListRuntimeResponse> filterByRuntimesInList(
+      Stream<org.pmiops.workbench.leonardo.model.ListRuntimeResponse> clustersToFilter,
+      List<String> runtimeNames) {
     // Null means keep all clusters.
     return clustersToFilter.filter(
-        cluster -> clusterNames == null || clusterNames.contains(cluster.getClusterName()));
+        cluster -> runtimeNames == null || runtimeNames.contains(cluster.getRuntimeName()));
   }
 
   @Override
   @AuthorityRequired(Authority.SECURITY_ADMIN)
   public ResponseEntity<List<ListClusterResponse>> deleteClustersInProject(
       String billingProjectId, ListClusterDeleteRequest clusterNamesToDelete) {
+    return ResponseEntity.ok(
+        deleteLeonardoRuntimesInProject(
+                billingProjectId, clusterNamesToDelete.getClustersToDelete())
+            .stream()
+            .map(leonardoMapper::toApiListClusterResponse)
+            .collect(Collectors.toList()));
+  }
+
+  @Override
+  @AuthorityRequired(Authority.SECURITY_ADMIN)
+  public ResponseEntity<List<ListRuntimeResponse>> deleteRuntimesInProject(
+      String billingProjectId, ListRuntimeDeleteRequest runtimesToDelete) {
+    return ResponseEntity.ok(
+        deleteLeonardoRuntimesInProject(billingProjectId, runtimesToDelete.getRuntimesToDelete())
+            .stream()
+            .map(leonardoMapper::toApiListRuntimeResponse)
```

Comment:
I was a bit confused since the class names are the same. Is it possible to prefix one of them like we do with `DbX` models?